GhostNetwork.Content.Api/Controllers/CommentsController.cs

```diff
@@ -45,6 +46,29 @@ public async Task<ActionResult<Comment>> CreateAsync([FromBody] CreateCommentMod
             return BadRequest(domainResult.ToProblemDetails());
         }
 
+        /// <summary>
+        /// Update comment content
+        /// </summary>
+        /// <param name="commentId">Existing comment ID</param>
+        /// <param name="content">New content</param>
+        /// <returns>Updated comment</returns>
+        [HttpPut("{commentId}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<Comment>> UpdateAsync([FromRoute] string commentId, [FromBody][Required] string content)
+        {
```

Comment:
Check if comment exists first of all by calling get by id directly in controller

Reply:
Thanks for the advice, added it.